docs/Upgrading.md

```diff
@@ -1,19 +1,90 @@
 # Upgrading
 
-- [Upgrading to v0.55.0-sumo-0](#upgrading-to-v0550-sumo-0)
-  - [`filter` processor: Drop support for `expr` language](#filter-processor-drop-support-for-expr-language)
-- [Upgrading to v0.52.0-sumo-0](#upgrading-to-v0520-sumo-0)
-  - [`sumologic` exporter: Removed `carbon2` and `graphite` metric formats](#sumologic-exporter-removed-carbon2-and-graphite-metric-formats)
-- [Upgrading to v0.51.0-sumo-0](#upgrading-to-v0510-sumo-0)
-  - [`k8s_tagger` processor: removed `clusterName` metadata extraction option](#k8s_tagger-processor-removed-clustername-metadata-extraction-option)
-    - [How to upgrade](#how-to-upgrade)
-  - [`sumologic` exporter: metadata translation: changed the attribute that is translated to `_sourceName` from `file.path.resolved` to `log.file.path_resolved`](#sumologic-exporter-metadata-translation-changed-the-attribute-that-is-translated-to-_sourcename-from-filepathresolved-to-logfilepath_resolved)
-    - [How to upgrade](#how-to-upgrade-1)
-- [Upgrading to 0.49.0-sumo-0](#upgrading-to-0490-sumo-0)
-  - [Several changes to receivers using opentelemetry-log-collection](#several-changes-to-receivers-using-opentelemetry-log-collection)
-  - [Sumo Logic exporter metadata handling](#sumo-logic-exporter-metadata-handling)
-    - [Removing unnecessary metadata using the resourceprocessor](#removing-unnecessary-metadata-using-the-resourceprocessor)
-    - [Moving record-level attributes used for metadata to the resource level](#moving-record-level-attributes-used-for-metadata-to-the-resource-level)
+- [Upgrading](#upgrading)
```

Comment:
I would skip this header in ToC

Reply:
An oversight of mine, this is an evildoing of a vscode extension which I have missed, I'm sorry.